projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -91,8 +91,6 @@ public static function load_features() {
 		require_once __DIR__ . '/features/wpcom-block-editor/functions.editor-type.php';
 		require_once __DIR__ . '/features/wpcom-site-menu/wpcom-site-menu.php';
 		require_once __DIR__ . '/features/wpcom-themes/wpcom-themes.php';
-		require_once __DIR__ . '/features/calypso-locale-sync/sync-wp-admin-to-calypso.php';
-		require_once __DIR__ . '/features/calypso-locale-sync/sync-calypso-to-wp-admin.php';
```

Comment:
Confidence check. Removing these shouldn't cause deploy time fatals on wpcom, right? 🤔

Reply:
I got this message on other PR when removing code:

** WARNING **
It is not safe to deploy the removal of a file along with the removal of the required_once() for this file.
Please undo the file removal and deploying it separately. Alternatively, you can create a new diff to remove the required_once() function and deploy that first.

Reply:
Oh, maybe we should just remove the requires in this PR to be on the safe side.